docs/documentation_guidelines.md

```diff
@@ -0,0 +1,11 @@
+# Documentation guidelines
+
+1. Each repository should contain an automatically updated index page in `docs` directory.
+2. Repository main `README.md` file should contain a link to the concerned repository index page, where all documentation contents are stored.
+3. Each `.md` file should contain at the beginning a title and a document description in a paragraph below. These two elements of `.md` file will be used to create index page.
+4. A `docs` directory under the repository root doesn't have to contain documentation related strictly to the one tool only. For example we can have a documentation describing a system comprised of multiple tools which together deliver some service for users.
+5. Documentation for specific tools and libraries should be maintained in the same directory as a tool or library.
+6. If a tool has more complex documentation structure it can have separate `docs` directory under it's root directory.
+7. Each document file should have a prefix that reflects the purpose of the document, for example: `how-to_`, `proposal_`, `standard_`, `architecture_`. If a document covers multiple topics for example `how-to` and `architecture` the prefix is not necessary.
```

Comment:
I'm on board with suffixes describing the purpose of the document.

However, I'm not sure how those "purposes" align with the [information types](https://kyma-project.io/community/guidelines/content-guidelines/01-content-strategy/#information-types) we have established for Kyma docs. We can certainly discuss whether the existing information types match modular Kyma (so far, discussions led to the conclusion that they do, perhaps with the addition of "architecture"). 

Also, mixing purposes in one document is not best practice in topic-based writing. 

In conclusion, this rule `7.` kind of competes with our content strategy; and while the strategy is absolutely open for discussion, we should indeed discuss any discrepancies first :) 

```suggestion
7. Each document file should have a prefix that reflects the purpose of the document, for example: `how-to_`, `proposal_`, `standard_`, `architecture_`. If a document covers multiple topics, such as `how-to` and `architecture`, the prefix is not necessary.
```

Reply:
Thanks Nina, it makes sense to have "one file per topic" so I decided to remove this part:

> If a document covers multiple topics for example `how-to` and `architecture` the prefix is not necessary.

Regarding prefixes, I am wondering if we have to strictly follow documentation guidelines for Kyma or new guidelines for modular Kyma since test-infra is not a part of a product.

Reply:
"Information types" listed [here](https://kyma-project.io/community/guidelines/content-guidelines/01-content-strategy#information-types) are good but we would be happy if we could  add more types like `architecture` or `standard`. What do you think, is that a crime? :)

Reply:
After discussion with Iwona I decided to get rid of document types  that are not mentioned in [information types](https://kyma-project.io/community/guidelines/content-guidelines/01-content-strategy/#information-types). I hope that we are now aligned 😄